using-timescaledb/distributed-hypertables.md

```diff
@@ -0,0 +1,124 @@
+# Distributed Hypertables
+
+>:WARNING: Distributed hypertables are currently in BETA and
+are not yet meant for production use. For more information, please
+[contact us][contact] or join the #multinode-beta channel in our 
+[community Slack][slack].
+
+Hypertables handle large amounts of data by breaking it up into
+smaller pieces (chunks), allowing operations to execute
+efficiently. When the amount of data is expected to be beyond what a
+single machine can handle, you can distribute the data chunks over
+several machines by using *distributed hypertables*.
+
+Distributed hypertables are otherwise similar to normal hypertables, but
+distributed hypertables will also spread a hypertable's chunks across multiple
+data nodes.  Distributed hypertables also typically use an additional "space
+partitioning" of data (e.g., by server or container ID, by device ID, by ticker
+symbol, etc.) so that an individual time interval is spread over all data nodes
+(although this is not required).  Thus to create a distribted hypertable, you
+need to have added at least one data node before starting.
```

Comment:
Suggestion:

> Thus to create a distributed hypertable, you need to have added at least one data node before starting. Distributed hypertables are typically also partitioned by an additional column, such as server or container ID, device ID, or ticker symbol. It is not strictly required, but can improve performance since individual time intervals are spread over several data nodes.